CONTRIBUTING.md

```diff
@@ -405,13 +436,14 @@ The minimum Node version supported by the project is stated under the `engines`
 [editor-config]: https://editorconfig.org
 [eslint]: https://eslint.org
 [prettier]: https://prettier.io
+[size-limit]: https://github.com/ai/size-limit
 [standard-version]: https://github.com/conventional-changelog/standard-version
-[ava]: https://github.com/avajs/ava
+[vitest]: https://vitest.dev/
 [siroc]: https://github.com/unjs/siroc
 [typescript]: https://www.typescriptlang.org
 [template]: https://github.com/prismicio/prismic-typescript-template
 [template-issue]: https://github.com/prismicio/prismic-typescript-template/issues/new/choose
 [changelog]: ./CHANGELOG.md
 [forum-question]: https://community.prismic.io
-[repo-issue]: https://github.com/prismicio/prismic-richtext/issues/new/choose
-[repo-pull-requests]: https://github.com/prismicio/prismic-richtext/pulls
+[repo-issue]: https://github.com/github_org_slash_github_repo/issues/new/choose
+[repo-pull-requests]: https://github.com/github_org_slash_github_repo/pulls
```

Comment:
Looks like a couple of copy-n-pastes forgot to be updated :)

Reply:
Nice catch!